src/dict.c

```diff
@@ -510,20 +510,23 @@ void dictRelease(dict *d)
 dictEntry *dictFind(dict *d, const void *key)
 {
     dictEntry *he;
-    uint64_t h, idx, table;
+    uint64_t h;
+    long idx_0, idx_1;
 
     if (dictSize(d) == 0) return NULL; /* dict is empty */
     if (dictIsRehashing(d)) _dictRehashStep(d);
     h = dictHashKey(d, key);
-    for (table = 0; table <= 1; table++) {
-        idx = h & DICTHT_SIZE_MASK(d->ht_size_exp[table]);
-        he = d->ht_table[table][idx];
-        while(he) {
-            if (key==he->key || dictCompareKeys(d, key, he->key))
-                return he;
-            he = he->next;
-        }
-        if (!dictIsRehashing(d)) return NULL;
+    idx_0 = h & DICTHT_SIZE_MASK(d->ht_size_exp[0]);
+    if (likely(d->rehashidx < idx_0)){
+        he = d->ht_table[0][idx_0];
+    } else {
+        idx_1 = h & DICTHT_SIZE_MASK(d->ht_size_exp[1]);
+        he = d->ht_table[1][idx_1];
+    }
+    while(he) {
```

Comment:
```suggestion
    while (he) {
```

Reply:
it seem not need a space between while and (he), because i see other place 
![image](https://user-images.githubusercontent.com/60819151/144748927-626adeca-4255-4927-84cf-961a1e2c345f.png)

Reply:
i know redis is a bit inconsistent in that area.
and actually, i just counted, there are slightly more cases that don't have a space after while than the ones who do.
unlike `for` in which it's clear that space is used more.

in that case, i don't mind.